.mvn/jvm.config

```diff
@@ -0,0 +1,4 @@
+-Xms256m
+-Xmx256m
+-XX:MaxPermSize=128m
+-Djava.awt.headless=true
```

Comment:
How did we arrive at these numbers?

Reply:
maven reports memory usage at the end of the build. Based on numbers, 64 MB sees too low. 128 MB should work, but I wanted to give it more memory as long as Travis build passes.